chapters/equations.tex

```diff
@@ -817,11 +817,16 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
 \lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
-automatically such that the resulting system is structurally nonsingular.  In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user
-expects.  A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without
-informing the user.  For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly
-used on these variables, do not have an effect on the simulation.
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.
+A tool may add or remove initial equations automatically such that the resulting system is structurally nonsingular.
+In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user expects.
+\begin{itemize}
+\item A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without informing the user.
+For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly used on these variables, do not have an effect on the simulation.
+\item A start-value that is not fixed may be seen as fixed with a message/warning.
```

Comment:
I think _treated as_ sounds better, and I believe we should stick with "diagnostics" for consistency (or change to message/warning also in other places):
```suggestion
\item A \lstinline!start!-attribute that is not fixed may be treated as fixed with a diagnostics.
```